### ISSUE TITLE
// in comments doesn't work for images

### ISSUE THREAD

**Issue by hartbit** [schibsted/layout#50]
```xml
<UITabBarController>
    <UIViewController
        title="Tab"
        tabBarItem.image="// Image-Normal"
        tabBarItem.selectedImage="// Image-Selected"
        xml="Other.xml"
    />
</UITabBarController>
```

**Comment by nicklockwood** [comment]
Images are treated as strings, so you'd need to wrap the comment in { }.

I agree this is awkward, but since an image name can be a path, it could plausibly start with a /. // is fairly unlikely though, so perhaps I can special-case it.

**Comment by hartbit** [comment]
Okay, understood :) I think special casing it would be worth it!

**Comment by nicklockwood** [comment]
This is now supported on the develop branch. See README for details.

**Comment by nicklockwood** [comment]
Fixed in 0.4.25

**Comment by hartbit** [comment]
👍
